server/mscalendar/subscription.go

```diff
@@ -81,17 +81,16 @@ func (m *mscalendar) RenewMyEventSubscription() (*store.Subscription, error) {
 		return nil, err
 	}
 
-	storedSub, err := m.Store.LoadSubscription(m.actingUser.Settings.EventSubscriptionID)
-	if err != nil {
-		return nil, err
+	var subToReturn *store.Subscription
```

Comment:
Maybe I am missing something, but I do no see where subToReturn is assigned any value.